packages/web/src/hooks/useImageSize.test.tsx

```diff
@@ -11,7 +11,7 @@ const TestComponent = (
   const dispatch = () => {}
   // @ts-ignore
   const image = useImageSize({ ...props, dispatch })
-  return <div>{image ?? 'nothing'}</div>
+  return <div>{image() ?? 'nothing'}</div>
```

Comment:
is this a bugfix? seems like the `useImageSize` overloads returning different but similar types could get confusing

Reply:
yeah i think this could be an error for sure, but also its just in a test so im kinda cool with it. looking at the code it was like, if dispatch is passed in, return a function, so potentially this is right, but also didn't care that much given test!

Reply:
yeah more concerned with the types being confusing for actual usages. If this bug can slip by in a test, we might make the same mistake for realsies

Reply:
yeah good call, let me dig into the intracies of this function rq

Reply:
Sweet. I'm assuming the reason for the function is when you want the calculation of the value deferred as late as possible, but idk

Reply:
@amendelsohn updated!